src/main/java/com/netchill/services/movie/MovieService.java

```diff
@@ -14,52 +13,33 @@
 @Singleton
 public class MovieService {
     private MovieDao movieDao;
-    private MovieDbApiClient movieDbApiClient;
+    private MovieApiService movieApiClient;
     private ConfigurationService configurationService;
 
     @Inject
-    private MovieService(MovieDao movieDao, MovieDbApiClient movieDbApiClient, ConfigurationService configurationService) {
+    private MovieService(MovieDao movieDao, MovieApiService movieDbApiClient, ConfigurationService configurationService) {
         this.movieDao = movieDao;
-        this.movieDbApiClient = movieDbApiClient;
+        this.movieApiClient = movieDbApiClient;
         this.configurationService = configurationService;
     }
-    public void updateImageUrl(List<Production>  prod) {
-        String imgKey;
-        String IMG_URL = this.configurationService.getImageBaseUrl();
-        for(Production element:  prod){
-            imgKey = element.getBackdrop_path();
-            if (imgKey != null) {
-                element.setBackdrop_path(IMG_URL + imgKey);
-            }
 
-            imgKey = element.getPoster_path();
-            if (imgKey != null) {
-                element.setPoster_path(IMG_URL + imgKey);
-            }
-        }
+    public Production getMovieById(Long movieId) {
+        return this.movieApiClient.getMovieById(movieId);
```

Comment:
si tu mets un id qui n'existe pas ça fait quoi ?

Reply:
Je sais que la réponse de TMDB c'est : 
`{
    "success": false,
    "status_code": 34,
    "status_message": "The resource you requested could not be found."
}`

J'imagine qu'étant donné que c'est pas compatible avec l'objet Production, ça renvoie un null ?

Reply:
ok met un todo tester avec un id qui n'existe pas